lib/core/src/display/shape/text.rs

```diff
@@ -94,6 +94,12 @@ impl TextComponent {
         }
     }
 
+    pub fn navigate_cursors(&mut self, step:Step, selecting:bool, fonts:&mut Fonts) {
```

Comment:
I relly don't like the idea that a font-map is involved in a cursor navigation operation. It seems to me like two concepts which should not mix. What is the reason that `fonts` have to be used here? Btw, I would find a better name for `Fonts`, like a `FontRegistry`.

Reply:
It's used in navigating line up/down. To find what column in another line have x position near current cursor we must know the width of each character. And that is in font info